ImageCapture/Image_Preprocessing.py

```diff
@@ -0,0 +1,73 @@
+import cv2
+import numpy as np
+
+def resize_image(image, size=(640, 640)):
```

Comment:
What is the type of image?

Reply:
Numpy array for OpenCV.
When the 3 features of our detection algorithm are finalized, I'm going to just use the file path to import image and convert it using openCV to be compatible with preprocessing and the model instance later as well. 
It can be done simply by: 
image = cv2.imread('path to image")